src/models/converters/transactions/mod.rs

```diff
@@ -198,15 +202,20 @@ impl MultisigTransaction {
         self.confirmations_required.unwrap_or(threshold)
     }
 
-    fn missing_signers(&self, owners: &Vec<String>) -> Vec<String> {
+    fn missing_signers(&self, owners: &Vec<String>) -> Vec<AddressEx> {
         self.confirmations.as_ref().map_or_else(
-            || owners.to_owned(),
+            || {
+                owners
+                    .iter()
+                    .map(|owner| AddressEx::address_only(&owner))
+                    .collect()
```

Comment:
I wanted to suggest to have a `From` implementation for `Vec<&str>` into `Vec<AddressEx>` for these sort of cases, but doesn't seem easily possible. Just as a general comment, unrelated to the PR, maybe we should start leveraging more `From` as it seems to be more idiomatic for these sort of situations.

Reply:
Although a disadvantage of `From` is  that we can't pass dependencies in addition to the `struct` instance to the function. And it's also not `async`. For simple use case should be fine though.